WORKING_GROUPS.md

```diff
@@ -434,6 +434,37 @@ Responsibilities include:
   backporting changes to these branches.
 * Define the policy for what gets backported to release streams.
 
+### [Security](https://github.com/nodejs/security-wg)
+
+The Security Working Group manages all aspects and process linked to security for Node.js.
+
+Responsibilities include:
+* Define and maintain security policies and procedures for:
+  * the core Node.js project
+  * other projects maintained by the Node.js Foundation technical group
+* Work with the node security project to bring community vulnerability data into
+  the foundation as a shared asset.
+* Set up processes and procedures and follow these to ensure the vulnerability
+  data is updated in an efficient and timely manner. For example, ensuring there
+  are well documented processes for reporting vulnerabilities in community
+  modules.
+* Work to set a high standard for the Node.js project. Possibly efforts could
+  include penetration testing, security reviews etc, review guidelines, coding
+  standards etc.
+* Review and recommend processes for handling of security reports (but not the
+  actual handling of security reports, which are reviewed by a group of people
```

Comment:
nit: Maybe it's more clear to change `handling ` -> `administration` ? Maybe not. 👍